recipes/pyfinance/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pyfinance" %}
+{% set version = "1.2.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e4f723b5e6e399baf1938031def0f9f7de7afccf990d19a37fb74b8093a43c4b
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.*
```

Comment:
you can drop the ``.*``

Reply:
But it is not necessary to change, just for the next time
the ``.*`` is not necessary in this case

Reply:
got it thanks, so just `python >=3`?